dvc/remote/base.py

```diff
@@ -546,7 +558,7 @@ def download(
             )
         except Exception:
             msg = "failed to download '{}' to '{}'"
-            logger.exception(msg.format(from_info, to_info))
+            logger.warning(msg.format(from_info, to_info))
```

Comment:
I think we should change this log to warning. If it does not block command execution, we should not log it as error and wrap it in messages pointing to our help page, especially that download error might be of various reasons.

Reply:
@pared But on warning we won't see info about the original error. https://github.com/iterative/dvc/blob/master/dvc/logger.py#L60 which is a problem. I would rather have an `exception` here, which prints the error and the cause.